src/alchemlyb/parsing/amber.py

```diff
@@ -89,8 +93,7 @@ def skip_after(self, pattern):
                 break
         return Found_pattern
 
-    def extract_section(self, start, end, fields, limit=None, extra='',
-                        debug=False):
```

Comment:
If a kwarg is removed, it changes the API in a breaking manner. This needs an entry in CHANGES under "Changes" and a versionchanged here. (As soon as we hit 1.x, we cannot do such a thing until we get to 2.0).

Reply:
The "problem" is that the `debug` kwarg was in the method definition, but was never referenced inside the method, so it never did anything in the first place. Do I need to state in the changelog that an unutilized kwarg was removed?

Reply:
no, if it never did anything then I am ok to just drop it; we _could_ mention it as a fix in CHANGES but let's not ...